File: PHP/Figures exercise/figures/square.php
<?php
 class Square extends FigureGeometric {

    private float $side;
    private float $calculate;
    
    public function __construct(float $side){
        parent::_construct("Square");
        $this-> side= $side;
    }
    public function getSide():float{
        return $this-> side;
    }
    public function calculateArea():float{
        $this->calculate = $this->getSide()* $this->getSide();
        return $this->calculate;
    }
}

File: JAVA/Figures exercise/com/figures/Square.java
package com.figures;

public class Square extends FigureGeometric {
    private int side;
    private double calculate;

    public  Square(int side) {
        super("Square", "Cuadrado");
        this.side = side;
    }
    public int getSide() {
        return this.side;
    }
    public double calculateArea() {
        this.calculate = this.getSide() * this.getSide();
        return this.calculate;
    }
    
}